modules/lang-painless/src/main/java/org/elasticsearch/painless/Definition.java

```diff
@@ -770,10 +770,10 @@ public Definition(List<Whitelist> whitelists) {
         }
 
         // precompute runtime classes
-        for (String painlessStructName : structsMap.keySet()) {
-            Struct painlessStruct = structsMap.get(painlessStructName);
+        for (Map.Entry<String, Struct> entry : structsMap.entrySet()) {
+            Struct painlessStruct = structsMap.get(entry.getKey());
 
-            if (painlessStruct.name.equals(painlessStructName) == false) {
+            if (!painlessStruct.name.equals(entry.getKey())) {
```

Comment:
Please revert to `== false`. This is used throughout the Elasticsearch codebase as a more visually abrupt form of negation to not be easily overlooked.

Reply:
Done